linux/user-and-file-management/linux-intro/linux-common-terminology.md

```diff
@@ -0,0 +1,57 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+
+---
+
+# Common Terminology
+
+---
+## Content
+
+
+Let's go over a few terms that you'll see a lot in this course.
+
+Linux comes with an improved version of the shell, which is called **bash**[1]. From now on, we'll be using these terms interchangeably.
```

Comment:
bash also refers to the programming language used to interpret the code within the shell
wondering if we should make this mention

Reply:
Hmmm, I feel like it might be too confusing this early on?

Reply:
I'll add a note like:

> 💡 Fun fact: **bash** also refers to the programming language used to interpret the code within the shell.